tools/rna_tools/sortmerna/sortmerna.xml

```diff
@@ -131,6 +165,56 @@ sortmerna --version 2>&1|grep 'SortMeRNA version'
         <param name="log" type="boolean" checked="False" truevalue="--log" falsevalue="" label="Generate statistics file"
                help="Generates statistics for the rRNA content of reads, as well as rRNA subunit distribution. (--log)">
         </param>
+	<conditional name="report">
+            <param name="report_type" type="select" display="radio" label="Parameters for filtering and read mapping" help="">
```

Comment:
I guess the IUC would recommend a normal select box here:
http://galaxy-iuc-standards.readthedocs.org/en/latest/best_practices/tool_xml.html

I admit it's not clear and I'm fine with a radiobox if this increases usability.